### ISSUE TITLE
Forest polygons

### ISSUE THREAD

**Issue by wipfli** [onthegomap/planetiler#104]
How can I make a custom layer with only forest polygons? I tried to figure it out from the base profile but failed...

**Comment by msbarry** [comment]
Does the base profile have what you want you just want to filter out all of the layers besides land use? Or do you want to make a new overlay mbtiles with a forest layer from scratch?

**Comment by wipfli** [comment]
I have a MapLibre GL JS style which comes close to the above Swiss 1:50'000 map. It uses OpenMapTiles. This style does not use all information that OpenMapTiles provides, e.g., it does not render `landuse: farmland`. I would like to customize the planetiler profile now and remove information that is not needed. An other example is the `transportation_name` layer which makes up over 20 percent of the full data. You see in the above map that road names are not rendered. For my ballooning map this is fine, and to save space, I would like to remove this layer. Removing full layers is possible with `--exclude_layers` I think.

**Comment by msbarry** [comment]
Yep, you can use `--exclude-layers` and `--only-layers` to filter-out entire layers, but I agree it would be nice to have more fine-grained control and be able to opt into/out of individual features within those layers or tags on those features.

It would be really cool would be to give it a maplibre-gl style json and have it figure out the minimal set of layers/features/tags to emit, although that could also be done as a 2-step process where planetiler takes the layer/feature/tag exclusions in some format, and another tool generates those exclusions from a style json.

Another thought I had around configurable schemas would be for your maplibre style spec to select features and tags directly from raw OSM data, and planetiler would produce intermediate tiles that just supports that style.
